docs/contracts/interfaces.rst

```diff
@@ -6,12 +6,14 @@
 Interfaces
 **********
 
-Interfaces are similar to abstract contracts, but they cannot have any functions implemented. There are further restrictions:
+Interfaces are similar to abstract contracts, but they cannot have any functions or modifiers implemented.
```

Comment:
If you say "modifiers implemented" here, it looks like it is OK to declare them.

Reply:
There's `They cannot declare modifiers` below. I wanted to emphasize that they cannot be defined either.

Reply:
I reworded it. How about now?